File: Class 1/findTwoSum.js
const nums = [3, 2, 4];
const target = 6;

var findTwoSum = function (nums, target) {
  let sum;

  for (let i = 0; i < nums.length; i++) {
    for (let j = 1; j < nums.length; j++) {
      sum = nums[i] + nums[j];
      if (sum === target) {
        return [i, j];
      }
    }
  }
};

console.log(findTwoSum(nums, target));

File: Class 1/findTwoSum.py
nums = [2,7,11,15]
target = 9

class Solution(object):
    def twoSum(self, nums, target):

      for i in range(len(nums)):
         for j in range(i+1, len(nums)):
            sum = nums[i] + nums[j]
            if sum == target:
                return [i,j]
        
print(Solution.twoSum(nums, target))